common/retrier/src/lib.rs

```diff
@@ -54,11 +55,25 @@ pub fn fixed_retry_strategy(delay_ms: u64, tries: usize) -> impl Iterator<Item =
     FixedDelay::new(delay_ms).take(tries)
 }
 
+pub fn exp_retry_strategy(
```

Comment:
share we re-order these fns to make exp_retry and fixed_retry are grouped by each other?

Reply:
I am not sure why we prefer this order over some other :)

Reply:
hmm, I thought it will be easier to read to group struct + impl + Iterator together but its definitely only personal preference.

Reply:
There are different styles in rust, there is no really guideline for libra as far as I am aware

Arbitrary choice I made here is to group blocks by type, e.g. struct delcaration first, then implementations, then trait implementations

Reply:
this makes sense too, I will keep this style consistency in the future changes